### DISCUSSION TITLE
Read/write CHROOT help message

### DISCUSSION THREAD

**Discussion by Sithuk** [zbm-dev/zfsbootmenu#428] [General]
tldr; add a help message on entry into read only chroot to remind the user to import the pool read/write first (Ctrl + W) if permanent changes are to be made.

Zfsbootmenu helpfully offers the ability to enter a read/write chroot to fix a system.

Zfsbootmenu loads the zfs pool(s) read only by default. I expect the intent of read only is to reduce the risk of accidental damage to the pool content, which makes sense. A shortcut for entering chroot is shown on the main screen (CTRL + J). There is no shortcut for importing read/write on the main screen. A user may not be aware of the read/write functionality.

There are some help messages shown when entering a read only chroot. It might be helpful to add a note to remind a user that zfsbootmenu offers the ability to import a pool read/write if permanent changes need to be made.

**Comment by zdykstra** [comment]
https://github.com/zbm-dev/zfsbootmenu/pull/429 is our first try at solving this.
